### ISSUE TITLE
SearchCenterPanel and GXP_QueryPanel quirks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [heron-mc/heron-mc#279]
```
There are some quirks in 'GXP_QueryPanel.js':
1. Missing i18n strings (heron)
2. If there are no WFS layers defined the code runs in an GXP store error.
3. A query can be done only once the app is started - if you start the query a 
second time, no result is reached - you must start the app again for the next 
query!?
4. When a great amount of result data is transfered to the client, it seems as 
if it is frozen - it would be nice to have a 'Load data' message.
- GXP i18n defect: the filter box is not translated when 
'../ux/gxp/git/src/script/locale/xx.js' is inluded.
```

Original issue reported on code.google.com by `wolfram.winter` on 17 Sep 2013 at 10:54
